rasa/shared/core/events.py

```diff
@@ -1534,7 +1537,13 @@ def __str__(self) -> Optional[Text]:
 
     def __hash__(self) -> int:
         """Returns unique hash for event."""
-        return hash(self.__members__())
+        return int(self.fingerprint(), 16)
+
+    def fingerprint(self) -> Text:
```

Comment:
Had to override `Event.fingerprint()` because a [story conflict test ](https://github.com/RasaHQ/rasa/blob/3f3c8830a99278578ea687bdfc3645d7c368e86b/tests/core/training/test_story_conflict.py#L139) relied on a constant hash for ACTION_UNLIKELY_INTENT ([here ](https://github.com/RasaHQ/rasa/blob/3f3c8830a99278578ea687bdfc3645d7c368e86b/rasa/core/training/story_conflict.py#L248)and [here](https://github.com/RasaHQ/rasa/blob/3f3c8830a99278578ea687bdfc3645d7c368e86b/rasa/core/training/story_conflict.py#L254)).